config/locales/en.yml

```diff
@@ -780,11 +780,17 @@ en:
       destroy: Are you sure you want to deactivate this business unit?
       team_members: Team members
       new_team_member: New team member
+    directorate_detail:
+      destroy: Are you sure you want to deactivate this directorate?
     edit:
       edit: Edit %{team_type}
     new:
       new: New %{team_type}
     destroyed: This team has now been deactivated
+    error: To deactivate this team you need to first deactivate all members.
+    deactivate_info:
```

Comment:
See comment above: the message gets displayed for all errors, not just active children.